client/resource_group/controller/controller.go

```diff
@@ -556,52 +559,60 @@ func newGroupCostController(
 func (gc *groupCostController) initRunState() {
 	now := time.Now()
 	gc.run.now = now
-	gc.run.lastRequestTime = now
+	gc.run.lastRequestTime = now.Add(-defaultTargetPeriod)
```

Comment:
Why not sub?

Reply:
`Sub` is used to get the duration between two `Time`s